changelogs/changelog.yaml

```diff
@@ -88,3 +88,28 @@ releases:
     fragments:
     - 44-plugin-doc-fixes.yaml
     release_date: '2020-06-23'
+  1.0.2:
+    changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in iosxr_config work when module
+        alias is used (https://github.com/ansible-collections/cisco.iosxr/pull/63).
+      - Makes sure that docstring and argspec are in sync and removes sanity ignores
+        (https://github.com/ansible-collections/cisco.iosxr/pull/62).
+      - Update docs after sanity fixes to modules.
+      minor_changes:
+      - Add changelogs for previous releases (https://github.com/ansible-collections/cisco.iosxr/pull/47)
+    fragments:
+    - 47-add-change-log-1.0.0.yaml
+    - handle_src_backup_with_module_alias.yaml
+    - remove_ignores_for_sanity.yaml
+    - update_docs.yaml
+    release_date: '2020-08-07'
+  1.0.3:
+    release_date: '2020-08-07'
```

Comment:
Is there anything at all that would make sense to have as a release summary for 1.0.3?

Reply:
so it was some doc update, would that work?

Reply:
So something like https://github.com/ansible-collections/ansible.netcommon/blob/main/changelogs/changelog.yaml#L156? We should just probably mention _something_ about why another release was needed, even if it was an "oops, forgot <X>" release.